doc/conf.py

```diff
@@ -405,8 +405,9 @@ def __getattr__(cls, name):
 # inheritance_diagram graphviz attributes
 inheritance_node_attrs = dict(color='lightskyblue1', style='filled')
 
-from directives import UsageDetails
-
+from directives import UsageDetails, CustomGalleryItemDirective
 
 def setup(app):
+    app.add_directive('customgalleryitem', CustomGalleryItemDirective)
     app.add_directive("usagedetails", UsageDetails)
+    app.add_stylesheet('xanadu_gallery.css')
```

Comment:
@josh146 not sure if that is needed?

Reply:
I think it is - this is the file that determines the look and style of the cards.

Reply:
Oh nice, I had forgotten I had already added the usage details role!